internal/impl/mysql/input_mysql_stream.go

```diff
@@ -226,14 +226,16 @@ func (i *mysqlStreamInput) Connect(ctx context.Context) error {
 	// canalConfig.Logger
 
 	for _, table := range i.tables {
-		canalConfig.IncludeTableRegex = append(canalConfig.IncludeTableRegex, regexp.QuoteMeta(table))
+		canalConfig.IncludeTableRegex = append(
+			canalConfig.IncludeTableRegex,
+			"^"+regexp.QuoteMeta(i.mysqlConfig.DBName+"."+table)+"$",
```

Comment:
Another bug that was caught, this is database and table, not just table. We only support a single database right now however.

someday we might support regex here...